server.js

```diff
@@ -1,29 +1,92 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import mongoose from "mongoose";
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
+const listEndpoints = require("express-list-endpoints");
```

Comment:
Super nice to list the endpoints!

Reply:
Nice functionality but... You shouldn't mix  2 methods of importing external modules. Choose between `import` and `require` syntax and be consistent. 😸